src/js/common/schemaform/helpers.js

```diff
@@ -18,6 +16,19 @@ function createFormPageList(formConfig) {
     }, []);
 }
 
+export function createPageListByChapter(formConfig) {
```

Comment:
Was confused that this is used in another PR, but not in this one, even though it was added here.